Shop.API/Controllers/GameController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Shop.BLL.Common.DataTransferObjects.Games;
+using Shop.BLL.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Shop.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class GameController : ControllerBase
+    {
+        private readonly IGameService _gameService;
+
+        public GameController(IGameService gameService)
+        {
+            _gameService = gameService;
+        }
+
+        [HttpGet]
+        public async Task<IEnumerable<GameResponseDto>> GetGames(CancellationToken cancellationToken)
```

Comment:
it is better to return ActionResult like Task<ActionResult<IEnumerable<GameResponseDto>>> and then write Ok(await _gameService.GetGamesAsync(cancellationToken))